.eslintrc.js

```diff
@@ -1111,7 +1111,10 @@ module.exports = {
       },
       overrides: [
         {
-          files: ['x-pack/plugins/security_solution/**/*.{js,mjs,ts,tsx}'],
+          files: [
+            'x-pack/plugins/security_solution/**/*.{js,mjs,ts,tsx}',
+            'x-pack/plugins/cases/public/**/*.{js,mjs,ts,tsx}',
```

Comment:
I think we should add the whole cases folder to include the `server` folder. Example: `x-pack/plugins/cases/**/*.{js,mjs,ts,tsx}`

Reply:
For review purposes, I'd like to keep this PR as is, in terms of scope. I just checked and including the entire cases folder would cause another 590 rule violations. I'm sure it'd be review-friendlier to include/fix those in a follow-up PR.

Reply:
Ok! Make sense 🙂.